python/dgl/nn/pytorch/gt/degree_encoder.py

```diff
@@ -53,36 +58,32 @@ def __init__(self, max_degree, embedding_dim, direction="both"):
             )
         self.max_degree = max_degree
 
-    def forward(self, g):
+    def forward(self, degrees):
         """
         Parameters
         ----------
-        g : DGLGraph
-            A DGLGraph to be encoded. Graphs with more than one type of edges
-            are not allowed.
+        degrees : Tensor
+            If direction is ``both``, it should be the stack of in degrees and out degrees
+            of the batched graph with zero padding, of shape :math:`(2, B, N)`.
+            Otherwise, it can be the in degrees or out degrees of the batched graph
+            with zero padding, of shape :math:`(B, N)`, where :math:`B` is the
+            batch size of the batched graph, and :math:`N` is the maximum number of nodes.
```

Comment:
```suggestion
            If :attr:`direction` is ``both``, it should be stacked in degrees and out degrees
            of the batched graph with zero padding, a tensor of shape :math:`(2, B, N)`.
            Otherwise, it should be zero-padded in degrees or out degrees of the batched
            graph, a tensor of shape :math:`(B, N)`, where :math:`B` is the
            batch size of the batched graph, and :math:`N` is the maximum number of nodes.
```